random-beans/src/main/java/io/github/benas/randombeans/PopulatorImpl.java

```diff
@@ -108,11 +118,18 @@
                 }
             }
             return result;
-        } catch (Exception e) {
+        } catch (InstantiationError | Exception e) {
             throw new BeanPopulationException("Unable to generate a random instance of type " + type, e);
         }
     }
 
+    private <T> Class<?> randomConcreteSubTypeOf(final Class<T> type) {
+        List<Class<?>> concreteSubtypes = ReflectionsUtils.getConcreteSubTypesOf(type);
+        if (concreteSubtypes.isEmpty()) throw new NullPointerException("Unable to find a concrete sub type of type " + type);
```

Comment:
here, I think we can simply warn the user and silently ignore the field instead of making the whole population fail with NPE. What do you think?